src/app/pages/MainArticles/components/MainArticles/MainArticles.component.tsx

```diff
@@ -40,8 +40,7 @@ interface OwnProps {
   date: string;
   handleDate(): void;
 }
-const fallbackImage = require('@assets/images/logo_espeo.svg');
-
+const fallbackImage = '';
```

Comment:
hmm this need to be restored, probably i have accidently remove dit, becasue i was not sure how we solved loading svgs

Reply:
i remeber that you have on  one of your branches this svg loading solved

Reply:
**never** make such changes to components functionality just in order to get unit tests working